MatrixSDK/Aggregations/MXAggregatedEditsUpdater.m

```diff
@@ -109,7 +109,10 @@ - (MXHTTPOperation*)replaceTextMessageEvent:(MXEvent*)event
     else
     {
         compatibilityText = [NSString stringWithFormat:@"* %@", text];
-        compatibilityFormattedText = [NSString stringWithFormat:@"* %@", formattedText];
+        if (formattedText)
```

Comment:
Is it worth doing this check on the `Fail to parse reply event` branch and also checking for empty strings?